modules/static/protocolbuffers/gofeatures/rsync.incl

```diff
@@ -0,0 +1,4 @@
++ LICENSE
++ go_features.proto
```

Comment:
Doesn't this have a directory path associated with it?

Reply:
Added full import path and appears to work fine - https://github.com/bufbuild/modules/pull/572.